src/erlcloud_ec2.erl

```diff
@@ -1188,6 +1188,7 @@ extract_image(Node) ->
      {description, get_text("description", Node)},
      {root_device_type, get_text("rootDeviceType", Node)},
      {root_device_name, get_text("rootDeviceName", Node)},
+     {creation_date, erlcloud_xml:get_time("creationDate", Node)},
```

Comment:
sounds simple but I cannot find it in AWS API docs http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html.
could you please provide an example response and would be even better if you put into the tests